src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
 public class SalaryInfo {
+    private static final int DAY = 0;
+    private static final int MONTH = 1;
+    private static final int YEAR = 2;
+    private static final int WORK_DAY = 0;
+    private static final int EMP_NAME = 1;
+    private static final int WORK_HOURS = 2;
+    private static final int PER_HOUR = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate from = parseToLocalDate(dateFrom);
+        LocalDate to = parseToLocalDate(dateTo);
+        HashMap<String, Integer> workersSalary = getNamesOnHash(names);
```

Comment:
```suggestion
        Map<String, Integer> workersSalary = getNamesOnHash(names);
```
It is just a rule, it is better to use interface as the type